### ISSUE TITLE
Profile selection not possible in 4.5.3

### ISSUE THREAD

**Issue by mpavsic** [osmandapp/OsmAnd-iOS#3165]
Since the last update (to OsmAnd 4.5.3 on iOS) some profiles were automatically disabled and it is not possible to re-enable them (the toggle in the settings immediately switches back to off). Some of such problematic profiles are base profiles (eg. Walking) while some are custom profiles (eg. CycloRoute).

**Comment by dmpr0** [comment]
Hello! Thank you for the feedback. 
Could you please export all settings and profiles and send it to support@osmand.net so we can test it.
You can export all data in Menu – Settings – Local backup – Backup as file

**Comment by mpavsic** [comment]
Thank you for the reply, I've send the exported settings to the e-mail address provided.

**Comment by BKrnk1337** [comment]
I have the same problem. I already got in contact with the support. Reinstalling the app worked for me. Not the best "bug fix" but as I need the app and the functionality to switch profiles, it was the only one working for me.

**Comment by yuriiurshuliak** [comment]
We have tested your issue using the latest TestFlight version. 
The bug was successfully fixed. Kindly await the upcoming application updates. 

Here are the reproduction steps:

- We imported all your settings and profiles from this link: [[Export_13-10-23OSF](https://github.com/osmandapp/OsmAnd-iOS/files/12895044/Export_13-10-23.zip)]
- Navigate to Menu > Settings > App Profiles.
- Test the profiles by toggling them on and off. 


OsmAnd Version 4.5.4.1

https://github.com/osmandapp/OsmAnd-iOS/assets/127092082/bfc5a8ef-a648-4bc2-9e4a-86b1bc7fc872
